src/graph.rs

```diff
@@ -155,12 +155,12 @@ pub struct Build {
     /// Command line to run.  Absent for phony builds.
     pub cmdline: Option<String>,
 
+    /// Controls how dependency information is processed after compilation.
+    pub deps: Option<String>,
```

Comment:
Given this should only ever be "gcc" or "msvc", I think this should at least be an enum.  (I don't really understand why you need this, it seems the bool carried as much info as before?)

Reply:
I was thinking of keeping it open as an extension point, e.g. for python dependency inference, etc. Is this the wrong use of the `deps` field?

Reply:
It is, but even if we ever added some new mechanism, adding it to an enum is reasonable too.